README.md

```diff
@@ -44,17 +44,14 @@ The pattern of Service Binding has prior art in non-Kubernetes platforms.  Herok
   - [Binding Values as Environment Variables](#binding-values-as-environment-variables)
     - [Resource Type Schema](#resource-type-schema-3)
     - [Example Resource](#example-resource-3)
-  - [Synthetic Provisioned Service](#synthetic-provisioned-service)
+  - [Multi-Application Bindings](#multi-application-bindings)
```

Comment:
😂

Reply:
Oh diff algorithm.

Reply:
More so that I didn't re-create the ToC in the Multi-Application Bindings PR to avoid a conflict. I had a mental note to revisit after the first PR merged, but this will take care of it.

Reply:
I think we should revisit that title :)